accounts-db/src/storable_accounts.rs

```diff
@@ -46,6 +47,15 @@ impl<'a> AccountForStorage<'a> {
             AccountForStorage::StoredAccountInfo(account) => account.pubkey(),
         }
     }
+
+    pub fn take_account(&self) -> AccountSharedData {
+        match self {
+            AccountForStorage::AddressAndAccount((_pubkey, account)) => {
+                create_account_shared_data(&**account)
+            }
+            AccountForStorage::StoredAccountInfo(account) => create_account_shared_data(&**account),
```

Comment:
Can this be simplified?

```suggestion
            AccountForStorage::StoredAccountInfo(account) => create_account_shared_data(*account),
```